src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -44,6 +45,12 @@ public TelemetryClient(TelemetryConfiguration configuration)
             }
 
             this.configuration = configuration;
+
+            // If someone created configuration from scratch and forgot to initialize the channel then use the default.
```

Comment:
i see that this comment was from earlier code - please modify the comment as well.